src/sgame/sg_cmds.cpp

```diff
@@ -4383,6 +4383,7 @@ static const commands_t cmds[] =
 	{ "noclip",          CMD_CHEAT_TEAM,                      Cmd_Noclip_f           },
 	{ "notarget",        CMD_CHEAT | CMD_TEAM | CMD_ALIVE,    Cmd_Notarget_f         },
 	{ "pubkey_identify", CMD_INTERMISSION,                    Cmd_Pubkey_Identify_f  },
+	{ "putcon",          CMD_CHEAT,                           Cmd_PutCon_f           },
```

Comment:
« `// keep the list synchronized with the list in cg_consolecmds for completion.` »

Reply:
Sure. And how to have auto-completion enabled for the parameters? That would be almost more important to me.

Reply:
Done. Without the parameters.